docs/readmes/faq/faq_magma.md

```diff
@@ -103,6 +103,14 @@ This section lists some of the commonly asked questions related to Magma operati
 
 - <https://github.com/magma/magma/tree/master/docs/readmes/lte>
 
+### How can I check if a given subscriber is idle or connected?
+
+  - With the users IMSI you can fetch the state of a given subscriber:
```

Comment:
Include detail that you need to SSH into the gateway where the subscriber is connected.